### DISCUSSION TITLE
MediaElement volume will be lower when microphone in used together IOS  ?

### DISCUSSION THREAD

**Discussion by xufeitt** [CommunityToolkit/Maui#1235] [Q&A] [answered]
When microphone in used then MediaElement open video , volume will be lower and can't be volume up ?

anyone know why ?    IOS only      Android is OK.

**Comment by VladislavAntonyuk** [comment, marked as answer]
I think this is how iOS behaves. Similar to when you play music and then ask Siri for something. Your music still plays, but quieter.

**Comment by xufeitt** [comment]
Hi everyone 
I had solved this problem.
Key point is AVAudioSession

why ?   search it by yourself

How to code it in c#    
 https://learn.microsoft.com/zh-cn/dotnet/api/avfoundation.avaudiosession?view=xamarin-ios-sdk-12

Here  is my solution ,  add it to your record init   of ios

AVAudioSession.SharedInstance().Init();
 AVAudioSession.SharedInstance().SetCategory(AVAudioSessionCategory.PlayAndRecord, AVAudioSessionCategoryOptions.DefaultToSpeaker);
 AVAudioSession.SharedInstance().SetActive(true);

**Comment by CliffAgius** [comment]
Closed as answered
